src/styles/primeng-theme/utils/_theme-variables.scss

```diff
@@ -224,3 +224,6 @@ $inputswitch-disabled-thumb-color: $gray;
 $toast-padding: $basic-padding * 2;
 $toast-bgcolor: $white;
 $toast-border-radius: $border-radius;
+
+//Dot Edit Toolbar
+$dot-edit-toolbar-main-height: 70px
```

Comment:
shouldn't you update that variable on other places?

Reply:
We need to move that toolbar and not call it "edit" because is being use it in other places.